config/v1/types_infrastructure.go

```diff
@@ -144,6 +148,18 @@ type GCPPlatformStatus struct {
 	Region string `json:"region"`
 }
 
+// OpenStackPlatformStatus holds the current status of the OpenStack infrastructure provider.
+type OpenStackPlatformStatus struct {
+	// ApiIntIp is an IP address managed by the OpenStack provider
+	// backing the internal `api-int` record.
+	ApiIntIp string `json:"apiIntIp,omitempty`
+
+	// NodeDnsIp is the IP address for the internal DNS used by the
+	// nodes. Unlike the one managed by the DNS operator, `NodeDnsIp`
+	// provides name resolution for the nodes themselves.
+	NodeDnsIp string `json:"nodeDnsIp,omitempty`
```

Comment:
As above, "nodeDnsIp" in the godoc, and also consider casing of acronyms

Reply:
Fixed